ccmlib/scylla_node.py

```diff
@@ -1240,6 +1242,12 @@ def flush(self):
         self.nodetool("flush")
         self._wait_no_pending_flushes()
 
+    def get_release_major_scylla_version(self):
+        version = self.get_node_scylla_version()
+        find = re.search(r"^(\d+.\d+).*", version)
+
+        return find.groups()[0] if find else ''
```

Comment:
I think this interface could have been less clunky if this function returned the result of parse_version(), and also fail if it couldn't find the version (I don't see what's the point in returning '' and hiding the error).

Reply:
Agreed.

Also, the regular expression uses a `.` wildcard, and you probably want a `\.` instead.

You can refactor `_get_scylla_version` to reuse the regular it uses to match the PEP-440 conforming version part.
For example:
```python
>>> s='4.6.dev-0.20210412.d1ce1b971'
>>> pattern=r'^((\d+!)?\d+([.-]\d+)*([a-z]+\d*)?([.-](post|dev|rc)\d*)*)(-.*)$'
>>> m=re.match(pattern, s)
>>> m.group(1)
'4.6.dev'
```